pandas/core/frame.py

```diff
@@ -2865,16 +2865,29 @@ def to_html(
         justify: str | None = None,
         max_rows: int | None = None,
         max_cols: int | None = None,
-        show_dimensions: bool | str = False,
+        show_dimensions: bool | str | None = None,
```

Comment:
dont' you want these as `lib.no_default`? (e.g. to see if the user passed them)?

Reply:
probably, I've not really used `lib.no_default` before and a previous suggestion was to use `None`. Happy to investigate and switch if preferred.

Reply:
`None` is fine to use for this purpose, as long as it is not a valid value for the keyword